### ISSUE TITLE
Error Executing Task on client causing freeze/crash when respawning after /kill

### ISSUE THREAD

**Issue by LucilleTea** [jellysquid3/Sodium#502]
### Reproduction Steps
1. Be an OP in spectator mode on a dedicated server
2. Fly away from spawn
3. /kill then respawn
4. Repeat 2 and 3 until the game freezes

### Crash Report File
[Log which contains the error, from an earlier sodium 0.1.1 snapshot.](https://gist.github.com/LucilleTea/db99df3561b4f68e98951081a00bf654)
[Log which does not contain the specific error, but still froze in the same way from these reproduction steps, from the latest sodium commit.](https://gist.github.com/LucilleTea/34c0828af8a8c08bdd17506b19d5ae26)

### Additional Information
Not sure whether that error only showing up sometimes is a version difference, down to randomness, or etc. It looks to me like the relevant code in sodium hasn't been changed between these versions.

**Comment by LucilleTea** [comment]
Confirmed the freeze happens with only sodium loaded (latest build), and does not happen with sodium removed.
Unfortunately the error continues to not show in the log, even though the freeze happens in the same situation.
The freeze seemed to always happen on the third /kill in my targeted reproduction. Using a server view distance of 16.

**Comment by fsharpseven** [comment]
This also occurs in singleplayer, I managed to reproduce it at 3 view distance. The problem seems to be that every time a player does /kill, the HashTable of loaded chunks has more chunks added to it for some reason (visible in the SodiumChunkCache debug string). As you do more and more /kills the number of loaded chunks will steadily increase (if enough chunks aren't unloaded) until `loadChunkFromPacket` will attempt to exceed the capacity of the loaded chunk FixedLongHashTable. At least that's what I found. I'm not entirely sure as to _why_ these chunks are added.

Logfile:
[debug-1.log](https://github.com/jellysquid3/sodium-fabric/files/5888933/debug-1.log)
